api.planx.uk/gis/classifiedRoads.js

```diff
@@ -4,58 +4,59 @@ import "isomorphic-fetch";
-    //   ref https://labs.os.uk/public/os-data-hub-examples/os-features-api/wfs-example-intersects#maplibre-gl-js
-    const xml = `
-      <ogc:Filter>
-        <ogc:Intersects>
-        <ogc:PropertyName>SHAPE</ogc:PropertyName>
-          <gml:Polygon srsName="EPSG:4326">
-            <gml:outerBoundaryIs>
-              <gml:LinearRing>
-                <gml:coordinates>${req.query.geom}</gml:coordinates>
-              </gml:LinearRing>
-            </gml:outerBoundaryIs>
-          </gml:Polygon>
-        </ogc:Intersects>
-      </ogc:Filter>
-    `;
+  // Create an OGC XML filter parameter value which will select the road features (lines)
+  //   that intersect with the buffered site boundary (polygon) coordinates
+  //   ref https://labs.os.uk/public/os-data-hub-examples/os-features-api/wfs-example-intersects#maplibre-gl-js
+  const xml = `
+    <ogc:Filter>
+      <ogc:Intersects>
+      <ogc:PropertyName>SHAPE</ogc:PropertyName>
+        <gml:Polygon srsName="EPSG:4326">
+          <gml:outerBoundaryIs>
+            <gml:LinearRing>
+              <gml:coordinates>${req.query.geom}</gml:coordinates>
+            </gml:LinearRing>
+          </gml:outerBoundaryIs>
+        </gml:Polygon>
+      </ogc:Intersects>
+    </ogc:Filter>
+  `;
 
-    // Define WFS parameters object
-    const params = {
-      service: "WFS",
-      request: "GetFeature",
-      version: "2.0.0",
-      typeNames: "Highways_RoadLink", // sourced from OS MasterMap Highways Network, uniquely includes "RoadClassification" attribute
-      outputFormat: "GEOJSON",
-      srsName: "urn:ogc:def:crs:EPSG::4326",
-      filter: xml,
-      key: process.env.ORDNANCE_SURVEY_API_KEY || "",
-    };
+  // Define WFS parameters object
+  const params = {
+    service: "WFS",
+    request: "GetFeature",
+    version: "2.0.0",
+    typeNames: "Highways_RoadLink", // sourced from OS MasterMap Highways Network, uniquely includes "RoadClassification" attribute
+    outputFormat: "GEOJSON",
+    srsName: "urn:ogc:def:crs:EPSG::4326",
+    filter: xml,
+    key: process.env.ORDNANCE_SURVEY_API_KEY || "",
+  };
 
+  try {
     const url = `https://api.os.uk/features/v1/wfs?${new URLSearchParams(params).toString()}`;
     const features = await fetch(url)
       .then(res => res.json())
       .then(data => {
         if (!data.features?.length) return;
         
         // Filter out any intersecting roads that are not classified
-        const classifiedFeatures = data.features.filter((feature) => feature.properties["RoadClassification"] !== "Unclassified");
+        //   find all possible values on page 43 https://www.ordnancesurvey.co.uk/documents/os-mastermap-highways-network-roads-technical-specification.pdf 
+        //   XX in future consider doing this directly in XML using <ogc:PropertyIsNotEqualTo>
+        const classifiedFeatures = data.features.filter((feature) => !["Unclassified", "Not Classified", "Unknown"].includes(feature.properties["RoadClassification"]));
```

Comment:
I actually think this is clearer and more readable than a big XML filter now that there's more than one condition 👍